Formula/lnav.rb

```diff
@@ -40,8 +42,9 @@ def install
     system "./autogen.sh" if build.head?
     system "./configure", "--disable-dependency-tracking",
```

Comment:
```suggestion
    system "./configure", "--disable-dependency-tracking",
                          "--disable-debug",
                          "--disable-silent-rules",
```
Let's try to be consistent with how we build other formulae.

Reply:
Or we can use `*std_configure_args`, it includes the following: `--disable-debug`, `--disable-dependency-tracking`, `--prefix=#{prefix}`, `--libdir=#{lib}`